### ISSUE TITLE
Resizing the window crashes the game

### ISSUE THREAD

**Issue by moechakucha** [magistermaks/mod-glsl#6]
Crash report below.

```log
---- Minecraft Crash Report ----
// Hi. I'm Minecraft, and I'm a crashaholic.

Time: 2022-08-25 14:30:06
Description: Rendering screen

java.lang.IllegalArgumentException: Window 0x0 size out of bounds (max. size: 32768)
	at net.minecraft.class_276.method_1231(class_276.java:110)
	at net.minecraft.class_276.method_22596(class_276.java:69)
	at net.minecraft.class_276.method_1234(class_276.java:58)
	at net.darktree.glslmc.render.ScalableCanvas.resize(ScalableCanvas.java:29)
	at net.darktree.glslmc.render.impl.PanoramaShaderRenderer.draw(PanoramaShaderRenderer.java:99)
	at net.minecraft.class_766.redirect$bak000$glsl_render(class_766.java:529)
	at net.minecraft.class_766.method_3317(class_766.java:19)
	at net.minecraft.class_442.method_25394(class_442.java:253)
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_276.method_1231(class_276.java:110)
	at net.minecraft.class_276.method_22596(class_276.java:69)
	at net.minecraft.class_276.method_1234(class_276.java:58)
	at net.darktree.glslmc.render.ScalableCanvas.resize(ScalableCanvas.java:29)
	at net.darktree.glslmc.render.impl.PanoramaShaderRenderer.draw(PanoramaShaderRenderer.java:99)
	at net.minecraft.class_766.redirect$bak000$glsl_render(class_766.java:529)
	at net.minecraft.class_766.method_3317(class_766.java:19)
	at net.minecraft.class_442.method_25394(class_442.java:253)

-- Screen render details --
Details:
	Screen name: net.minecraft.class_442
	Mouse location: Scaled: (2147483647, 2147483647). Absolute: (1205.000000, 367.000000)
	Screen size: Scaled: (960, 505). Absolute: (1920, 1009). Scale factor of 2.000000
Stacktrace:
	at net.minecraft.class_757.method_3192(class_757.java:881)
	at net.minecraft.class_310.method_1523(class_310.java:1177)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, lambdabettergrass/default, continuity/default, continuity/glass_pane_culling_fix
Stacktrace:
	at net.minecraft.class_6360.method_36565(class_6360.java:49)
	at net.minecraft.class_310.method_1587(class_310.java:2348)
	at net.minecraft.class_310.method_1514(class_310.java:787)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.2, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Microsoft
	Memory: 4679158864 bytes (4462 MiB) / 5708447744 bytes (5444 MiB) up to 6014631936 bytes (5736 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz
	Identifier: Intel64 Family 6 Model 94 Stepping 3
	Microarchitecture: unknown
	Frequency (GHz): 3.19
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: NVIDIA GeForce GTX 1060 3GB
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 3072.00
	Graphics card #0 deviceId: 0x1c02
	Graphics card #0 versionInfo: DriverVersion=27.21.14.5671
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Virtual memory max (MB): 27568.76
	Virtual memory used (MB): 21399.98
	Swap memory total (MB): 11264.00
	Swap memory used (MB): 538.18
	JVM Flags: 6 total; -XX:+UseG1GC -XX:-UseAdaptiveSizePolicy -XX:-OmitStackTraceInFastThrow -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmn256m -Xmx5734m
	Fabric Mods: 
		architectury: Architectury 6.2.43
		bettercombat: Better Combat 1.0.13+1.19
			player-animator: Example Mod 0.0.5
			tinyconfig: TinyConfig 1.0.2
		cloth-config: Cloth Config v8 8.0.75
			cloth-basic-math: cloth-basic-math 0.6.1
		continuity: Continuity 2.0.1+1.19
		crawl: Crawl 0.11.0
			mm: Manningham Mills 2.3
		disable_custom_worlds_advice: Disable Custom Worlds Advice 3.0
		do-a-barrel-roll: Do a Barrel Roll 1.1.0+1.19
		environmentz: EnvironmentZ 1.2.3
		extremesoundmuffler: Extreme sound muffler 3.32_fabric-1.19.1
			fiber: fiber 0.23.0-2
		fabric-api: Fabric API 0.60.0+1.19.2
			fabric-api-base: Fabric API Base 0.4.11+e62f51a390
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.9+9ff28f4090
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.17+aeb40ebe90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.20+aeb40ebe90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.1+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.11+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.1.7+0c17ea9690
			fabric-commands-v0: Fabric Commands (v0) 0.2.28+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.34+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.2.3+aeb40ebe90
			fabric-convention-tags-v1: Fabric Convention Tags 1.1.1+7cd20a1490
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.5+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.1.8+a680b9b490
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.30+aeb40ebe90
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.18+9ff28f4090
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.28+aeb40ebe90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.28+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.21+aeb40ebe90
			fabric-item-api-v1: Fabric Item API (v1) 1.5.7+35a03c4390
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.28+aeb40ebe90
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.20+aeb40ebe90
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.18+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.1.2+aeb40ebe90
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.3+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.6+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.3+176380a290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.14+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.17+aeb40ebe90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.4+5eb68ef290
			fabric-networking-v0: Fabric Networking (v0) 0.3.21+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.0.11+aeb40ebe90
			fabric-particles-v1: Fabric Particles (v1) 1.0.10+aeb40ebe90
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.24+aeb40ebe90
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.10+aeb40ebe90
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.12+aeb40ebe90
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.20+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.14+aeb40ebe90
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.7+aeb40ebe90
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.22+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.10.16+aeb40ebe90
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.0.11+e62f51a390
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.6.1+aeb40ebe90
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.26+f05effaf90
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.2.13+9ff28f4090
			fabric-textures-v0: Fabric Textures (v0) 1.0.20+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.0.11+83a8659290
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.0+42d99c3290
		fabric-language-kotlin: Fabric Language Kotlin 1.8.2+kotlin.1.7.10
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.10
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.10
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.10
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.10
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.2
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.3
		fabricloader: Fabric Loader 0.14.9
		firstperson: FirstpersonModel 2.1.2
		forgeconfigapiport: Forge Config API Port 4.1.4
			com_electronwill_night-config_core: core 3.6.5
			com_electronwill_night-config_toml: toml 3.6.5
		glsl_panorama: GLSL Panorama 0.1.1
		iceberg: Iceberg 1.0.46
		idwtialsimmoedm: idwtialsimmoedm 0.1.2+1.19
		immersive_portals: Immersive Portals 2.2.0
			imm_ptl_core: Immersive Portals Core 2.2.0
			q_misc_util: QMiscUtil 2.2.0
		indium: Indium 1.0.7+mc1.19
		iris: Iris 1.2.6
			org_anarres_jcpp: jcpp 1.4.14
		java: OpenJDK 64-Bit Server VM 17
		lambdabettergrass: LambdaBetterGrass 1.3.0+1.19
		lambdynlights: LambDynamicLights 2.1.2+1.19
			pride: Pride Lib 1.1.2+1.19
			spruceui: SpruceUI 4.0.0+1.19
		legendarytooltips: Legendary Tooltips 1.2.6
		minecraft: Minecraft 1.19.2
		modmenu: Mod Menu 4.0.6
		neutral_animals: Neutral Animals 1.2.1
		notenoughanimations: NotEnoughAnimations 1.6.1
		primeval: Primeval 0.0.1
			cardinal-components-base: Cardinal Components API (base) 5.0.1
			cardinal-components-entity: Cardinal Components API (entities) 5.0.1
			incubus_core: Incubus Core 1.7.0
			satin: Satin 1.9.0
			trinkets: Trinkets 3.4.0
		roughlyenoughitems: Roughly Enough Items 9.1.528
			error_notifier: Error Notifier 1.0.9
		roughlyenoughresources: Roughly Enough Resources 2.6.0
		skinlayers: 3d Skin Layers 1.5.1-mc1.19.1
		sodium: Sodium 0.4.2+build.16
			org_joml_joml: joml 1.10.4
		telepistons: Telepistons 1.1.3
		waveycapes: WaveyCapes 1.2.1
		zoomify: Zoomify 2.2.0
			com_github_llamalad7_mixinextras: mixinextras 0.0.12
			dev_isxander_settxi_settxi-core: settxi-core 2.7.3
			dev_isxander_settxi_settxi-kotlinx-serialization: settxi-kotlinx-serialization 2.7.3
			settxi-gui-cloth-config: Settxi Gui (Cloth Config) 2.7.3
	Loaded Shaderpack: ComplementaryReimagined_r1.1.1.zip
		Profile: Custom (+7 options changed by user)
	Launched Version: Primevality
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: GeForce GTX 1060 3GB/PCIe/SSE2 GL version 3.2.0 NVIDIA 456.71, NVIDIA Corporation
	Window size: 1920x1009
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, Fabric Mods, lambdabettergrass/default, continuity/default, continuity/glass_pane_culling_fix
	Current Language: 简体中文 (中国大陆)
	CPU: 4x Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz
```

**Comment by magistermaks** [comment]
> java.lang.IllegalArgumentException: Window 0x0 size out of bounds (max. size: 32768)  

Did you resize the widow to 0x0 pixels? :thinking:

**Comment by moechakucha** [comment]
Nope, just windowed fullscreen with taskbar

**Comment by magistermaks** [comment]
ok i will try fixing this

**Comment by magistermaks** [comment]
Fixed version is now released as 0.1.2 (for 1.19.0, 1.19.1, 1.19.2)
